tests/completor/test_main.py

```diff
@@ -47,6 +47,13 @@
 /
 """
 
+WSEGINJV = """
+WSEGINJV
+-- Number    Cv_Inj      Primary_Ac      Secondary_Ac      Water_Rate_cutoff     Pressure_drop_cutoff
+    1        0.96        4.700e-04       8e-5              -150                  -0.5
```

Comment:
do we need to put the rate and pressure drop as negative? it is not a so user friendly or intepretable as negative?